### DISCUSSION TITLE
Numbers on the Map

### DISCUSSION THREAD

**Discussion by MoonshineSG** [Koenkk/zigbee2mqtt#11709] [General]
Just starting with z2m and I have this test setup... Can someone explain what the numbers on the different lines mean ? And is this considered acceptable connection speed/quality ? Thanks.

![2022-03-05_10-23-04](https://user-images.githubusercontent.com/687747/156864192-a0620b90-3826-4eb9-89b9-b7ca5787b0f8.jpg)

**Comment by lux73** [comment]
afaik all LQI above 30 works without Porblem - a LQI below 20 is considered as weak (and should be tweaked)

higher LQI = better (maximim is 255)

BUT i even realized that the LQI's shown are not reliable

**Reply by MoonshineSG** [reply]
but what is "X over Y" ? like the 80/72 between 2 nodes ? Or 252/99 between a Node and the Controller ?

**Reply by lux73** [reply]
Coordinator is marked as Star

Master Bedroom -> Coordinator LQI 252
Coordinator -> Master Bedroom LQI 99

Most Battery Devices has only Uplink to Coordinator

**Reply by MoonshineSG** [reply]
Oh! "up" and "down" have different LQI... I did not know that! Now it's clear...
